bi_view_editor/models/bve_view.py

```diff
@@ -96,7 +90,7 @@ def _get_field_list(fields_info):
                     view_fields.append(field_def)
             return view_fields
 
-        fields_info = json.loads(self._get_format_data(self.data))
+        fields_info = json.loads(self.data)
```

Comment:
Isn't the idea of using a fields.Serialized that it loads the json for you?

>>> cat._fields['testser'].type
'serialized'
>>> cat.write({'testser': {'a': 1}})
True
>>> type(cat.testser)
<type 'dict'>
>>> cat.testser
{u'a': 1}
```

Reply:
Thanks for your review @StefanRijnhart, it took me a while to understand that myself, but actually I'm only using the field for its escaping power, here is what happens:
1. In the client, the query is composed and `JSON.stringify`ed in order to send it to the server (I think the only way to send field values to the server is using strings);
2. In the server, the string is saved into the serialized field, it applies `json.dumps` on the string (then it is double-escaped);
3. When I access `self.data`, the serialized field itself `json.loads` it, but it is still a string, then I have to `json.loads` that *again* too

I recognize that it would be cleaner to escape the string only once, but I should override the `create` and intervene between steps 1 and 2 in order to load the field correctly (`json.load`ing that already).